ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-e2e.yaml

```diff
@@ -124,6 +124,42 @@ objects:
 
         mkdir -p "${HOME}"
 
+        # Install python-openstackclient which is currently required to run
+        # some tests (in parallel mode). See: https://github.com/kubernetes/kubernetes/blob/master/test/e2e/storage/drivers/in_tree.go#L956
+        cat > /etc/yum.repos.d/rdo-stein.repo <<-"EOF_CAT"
```

Comment:
From [CI][1]:

    /bin/bash: line 12: /etc/yum.repos.d/rdo-stein.repo: Permission denied

I'm not clear on why.

[1]: https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_release/4933/rehearse-4933-pull-ci-openshift-cluster-api-provider-openstack-master-e2e-openstack/1#0:build-log.txt%3A27